client/src/app/@theme/layouts/sample/sample.layout.scss

```diff
@@ -43,7 +43,7 @@
     height: 100vh;
     z-index: -1;
     will-change: transform;
-    background: url('/assets/images/logoBG.png') no-repeat center;
+    //background: url('/assets/images/logoBG.png') no-repeat center;
```

Comment:
Because we're on source control, these lines can be outright removed.

If this is the only use of this `logoBG.png` file, you may remove it as well.

Reply:
Sounds good. I have removed the line and the file.